### PR TITLE
Update PermissionsExample.java

### PR DESCRIPTION
**Pull request by byRoadrunner**
Added PermissionProvider examples

### CODE REVIEW

File: cloudnet-examples/src/main/java/de/dytanic/cloudnet/PermissionsExample.java [hunk 1]
```diff
@@ -15,17 +15,16 @@
...
-        PermissionPool permissionPool = CloudAPI.getInstance()
-                                                .getPermissionPool(); //Returns the permission pool with all permissions groups etc.
-        PermissionGroup permissionGroup = new Boolean(true) ? CloudAPI.getInstance()
-                                                                      .getPermissionGroup("Admin") : permissionPool.getGroups()
-                                                                                                                   .get("Admin"); //Options for got the permission group
+    //More options
+    public void handlePermissions()
+    {
+        PermissionPool permissionPool = CloudAPI.getInstance().getPermissionPool(); //Returns the permission pool with all permissions groups etc.
+        PermissionGroup permissionGroup = new Boolean(true) ? CloudAPI.getInstance().getPermissionGroup("Admin") : permissionPool.getGroups().get("Admin"); //Options for got the permission group
```
Comment (einrobin) [inline cloudnet-examples/src/main/java/de/dytanic/cloudnet/PermissionsExample.java]: "new Boolean(true)"?

**Review by GiantTreeLP — CHANGES_REQUESTED** [review]
The added lines are okay.
Please stick to the project-wide and new code style.
It might not be in the development branch just now, but you see that some parts are very questionable (like the curly braces being in the next line). We write Java not C.

Also, care to explain line 27?

File: cloudnet-examples/src/main/java/de/dytanic/cloudnet/PermissionsExample.java [hunk 2]
```diff
@@ -33,7 +31,7 @@ public void handlePermissions()
         CloudAPI.getInstance().updatePermissionGroup(permissionGroup); //Updates the permission group
     }
 
-    //Easier to use
+    //Easyer to use
```
Comment (GiantTreeLP) [inline cloudnet-examples/src/main/java/de/dytanic/cloudnet/PermissionsExample.java]: Really?
This was correct in the old version.

image

**Review by GiantTreeLP — CHANGES_REQUESTED** [review]

File: cloudnet-examples/src/main/java/de/dytanic/cloudnet/PermissionsExample.java (hunk 2 above)
Comment (byRoadrunner) [reply cloudnet-examples/src/main/java/de/dytanic/cloudnet/PermissionsExample.java]: uhm, oops xD
i´ll fix it

**Review by GiantTreeLP — APPROVED** [review]

**Pull request merged**